lib/containers/common.pm

```diff
@@ -123,6 +123,7 @@ sub test_container_image {
     assert_script_run("$runtime container create --name '$container' '$image' /bin/sh -c '$smoketest'");
     assert_script_run("$runtime container start '$container'");
     assert_script_run("$runtime container logs '$container' > '/var/tmp/container_$container'");
+    assert_script_run("$runtime wait '$container'", 90);
```

Comment:
I would also add a wait or sleep between removing a container and checking the status. I've seen some random failures because of doing things too quickly, but ok to do it in another PR.

Reply:
I will keep this for @grisu48. This is rather hotfix.

Reply:
I'll update this in another PR